__tests__/app.test.js

```diff
@@ -34,4 +34,36 @@ describe('nc-be-news-app', () => {
       expect(body.msg).toBe('Route not found!');
     });
   });
+
+  describe('GET /api/articles/:article_id', () => {
+    it('Should return an article object with the id given', async () => {
+      const { body } = await request(app).get('/api/articles/1').expect(200);
+
+      expect(body.article).toBeInstanceOf(Object);
+
+      expect(body.article.article_id).toBe(1);
+
+      expect(body.article).toEqual({
+        article_id: 1,
+        title: 'Living in the shadow of a great man',
+        topic: 'mitch',
+        author: 'butter_bridge',
+        body: 'I find this existence challenging',
+        created_at: '2020-07-09T20:11:00.000Z',
+        votes: 100,
+      });
+    });
+
+    it('Should return a 404 error if given wrong type for input id', async () => {
```

Comment:
would this be a 400 or a 404?
is it a bad request or is it just not found?

Reply:
Have sorted this out, is it okay to push now?